src/debugging/coreclr/dockerManager.ts

```diff
@@ -357,13 +386,65 @@ export class DefaultDockerManager implements DockerManager {
             permissions: 'ro'
         };
 
-        const volumes: DockerContainerVolume[] = [
+        let volumes: DockerContainerVolume[] = [
```

Comment:
>volumes [](start = 12, length = 7)

On macos, does the default volume sharing configuration include the new directories that we're now using?  I know that one pain point in VS for Mac is the necessity to configuration volume sharing to include the dotnet sdk directory...

Reply:
The default includes `/Users` (all of which these are under) so at least we're not adding a problem there. Users would still need to add the SDK folder though.